float64 position #optional
float64 velocity #optional
float64 acceleration #optional
---
string message
